### PR TITLE
ENG-3954: Add new env variables for handle resourcePath of assets in development 

### PR DESCRIPTION
**Pull request by Dv92**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,18 @@ Dev instances should be using the `.env.development.local` file while production
...
 a boolean used to determine whether the API calls will be against a real Entando Core or if they are just being mocked internally.
 
+#### `USE_MFE` (boolean, default: `false`)
+a boolean used to enable EPC's MFE rendering on AppBuilder.
+
+#### `USE_MFE_MOCKS` (boolean, default: `false`)
+a boolean used to enable the mocking system based on MSW for MFE's, usefull for development purpose.
+
+#### `USE_LOCAL_MFE` (boolean, default: `false`)
+a boolean used to reference the MFE static resources on the `/` instead of `entando-de-app/cmsresources`, this is usefull for testing with a local static server without infrastructure.
```
Comment (cecchisandrone) [inline README.md]: useful

File: README.md [hunk 2]
```diff
@@ -16,6 +16,18 @@ Dev instances should be using the `.env.development.local` file while production
 #### `USE_MOCKS` (boolean, default: `true`)
 a boolean used to determine whether the API calls will be against a real Entando Core or if they are just being mocked internally.
 
+#### `USE_MFE` (boolean, default: `false`)
+a boolean used to enable EPC's MFE rendering on AppBuilder.
+
+#### `USE_MFE_MOCKS` (boolean, default: `false`)
+a boolean used to enable the mocking system based on MSW for MFE's, usefull for development purpose.
```
Comment (cecchisandrone) [inline README.md]: useful

File: README.md [hunk 3]
```diff
@@ -16,6 +16,18 @@ Dev instances should be using the `.env.development.local` file while production
...
+
+#### `USE_LOCAL_MFE` (boolean, default: `false`)
+a boolean used to reference the MFE static resources on the `/` instead of `entando-de-app/cmsresources`, this is usefull for testing with a local static server without infrastructure.
+
+#### `USE_REMOTE_MFE` (boolean, default: `false`)
+a boolean used to reference the MFE static resources on `entando-de-app/cmsresources`, this is usefull for testing in remote cluster with a local AppBuilder instance.
```
Comment (cecchisandrone) [inline README.md]: useful

File: README.md [hunk 4]
```diff
@@ -16,6 +16,18 @@ Dev instances should be using the `.env.development.local` file while production
 #### `USE_MOCKS` (boolean, default: `true`)
 a boolean used to determine whether the API calls will be against a real Entando Core or if they are just being mocked internally.
 
+#### `USE_MFE` (boolean, default: `false`)
+a boolean used to enable EPC's MFE rendering on AppBuilder.
```
Comment (cecchisandrone) [inline README.md:20]: does this mean we can disable the feature and run AB without the need of any ECR or mocks, right?

Comment (Dv92) [reply README.md:20]: yes, that's the point

**Review by cecchisandrone — APPROVED** [review]

**Pull request merged**